home.html

```diff
@@ -1,53 +1,91 @@
 <!DOCTYPE html>
 <html>
-	<head>
```

Comment:
I have fixed the indenting of the home page which is why the code appears red here.